src/optimise/Optimise.jl

```diff
@@ -3,7 +3,7 @@ module Optimise
 using LinearAlgebra
 import ArrayInterface
 
-export train!, update!,
+export train!, step!, update!,
```

Comment:
```suggestion
export train!, optimstep!, update!,
```
This needs to be updated too.